sdk/cosmos/azure-cosmos/azure/cosmos/aio/cosmos_client.py

```diff
@@ -135,40 +141,39 @@ async def __aenter__(self):
     async def __aexit__(self, *args):
         return await self.client_connection.pipeline_client.__aexit__(*args)
 
-    async def close(self):
-        # type: () -> None
+    async def close(self) -> None:
         """Close this instance of CosmosClient."""
         await self.__aexit__()
 
     @classmethod
-    def from_connection_string(cls, conn_str, credential=None, consistency_level=None, **kwargs):
```

Comment:
Looks like we dropped optionally passing in a credential here?
We usually support that for cases where customers want to use a TokenCredential, in place of the key in the connection string. Agreed it would be unusual, so it can be hidden away in kwargs.